_script/legatus.py

```diff
@@ -0,0 +1,115 @@
+import os
+
+from git import Repo
+
+DEV = "dev"
+PROD = "prod"
+GOOGLE_REMOTE = "google"
+GITHUB_REMOTE = "origin"
+RELEASE_MESSAGE_PREFIX = "Release Sprint"
+MASTER_BRANCH = "master"
+
+# functions
+AUTH = "auth"
+BROKER = "broker"
+CALLBACK = "callback"
+CLAIM = "claim"
+DOCUMENT = "document"
+ENRICH = "enrich"
+NETWORK = "network"
+MGA = "mga"
+MAIL = "mail"
+PARTNERSHIP = "partnership"
+PAYMENT = "payment"
+POLICY = "policy"
+PRODUCT = "product"
+QUESTION = "question"
+QUOTE = "quote"
+RENEW = "renew"
+RESERVED = "reserved"
+RULES = "rules"
+SELLABLE = "sellable"
+TRANSACTION = "transaction"
+USER = "user"
+
+updatable_functions = [
+    AUTH,
+    BROKER,
+    CALLBACK,
+    CLAIM,
+    DOCUMENT,
+    ENRICH,
+    NETWORK,
+    MGA,
+    MAIL,
+    PARTNERSHIP,
+    PAYMENT,
+    POLICY,
+    PRODUCT,
+    QUESTION,
+    QUOTE,
+    RENEW,
+    RESERVED,
+    RULES,
+    SELLABLE,
+    TRANSACTION,
+    USER
+]
+changed_functions = [
+]
+sprint_number = 0
+dry_run = True
+
+
+def main():
+    print(r"""
+.____                                __                  
+|    |      ____     ____  _____   _/  |_  __ __   ______
+|    |    _/ __ \   / ___\ \__  \  \   __\|  |  \ /  ___/
+|    |___ \  ___/  / /_/  > / __ \_ |  |  |  |  / \___ \ 
+|_______ \ \___  > \___  / (____  / |__|  |____/ /____  >
+        \/     \/ /_____/       \/                    \/ 
+"""
+          )
+
+    repo = Repo(os.curdir)
+    git = repo.git
+
+    repo.remote(GITHUB_REMOTE).fetch(tags=True)
+    repo.remote(GOOGLE_REMOTE).fetch(tags=True)
+
+    created_tags = []
+
+    for function_name in changed_functions:
+        if function_name not in updatable_functions:
+            print(f"ERROR: {function_name} unknown function...skipping")
+            continue
+
+        tags = git.tag('--list', '--sort=-taggerdate', f'{function_name}/*.dev').splitlines()
+        dev_tag = tags[0]
+
+        git.checkout(dev_tag)
+
+        production_tag = dev_tag.replace(DEV, PROD)
+        if dry_run:
+            print(production_tag)
```

Comment:
Enhance message with something like:
`Tag to be created: {production_tag}`
`Dry run on, skipping tag creation...`